### PR TITLE
clock solved

### PR DESCRIPTION
**Pull request by krolich500grn**
- [DEMO LINK](https://krolich500grn.github.io/layout_stop-watch/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,7 +8,11 @@
...
 </head>
-<body>
-  <h1>Stop watch</h1>
+<body class="watch">
+  <div class="watch__container">
+    <div class="watch__sec-arrow"></div>
+    <div class="watch__min-arrow"></div>
+    <div class="watch__dot"></div>
+  </div>
 </body>
```
Comment (jstmpelowycz) [inline src/index.html:17]: missing one indentation for both `body` and `header`

Comment (jstmpelowycz) [reply src/index.html:17]: или не missing, линт же прошел

**Review by jstmpelowycz — APPROVED** [review]
работа бомба!

File: src/styles/blocks.scss/watch.scss [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    background-color: $sec-arrow-color;
+    border-radius: 50%;
+  }
+}
+
+@keyframes move {
+  0% {
+    transform: rotate(0);
+  }
+
```
Comment (Ihor-Karpyn) [inline src/styles/blocks.scss/watch.scss:53]:
```suggestion
```

**Review by Ihor-Karpyn — APPROVED** [review]

**Pull request closed without merging**